networktables/entry.py

```diff
@@ -12,6 +12,7 @@
 )
 
 from ntcore.value import Value
+from ntcore.support.compat import basestring
```

Comment:
Sorry, didn't see this. Because of #73 , this is no longer needed. Please rebase, thanks!